### PR TITLE
Allow for profile viewing

### PR DESCRIPTION
**Pull request by michaelwehrley**
* When participant isn't part of an active group catch profile error

[Finishes: #121680805]

### CODE REVIEW

File: spec/dummy/app/assets/config/manifest.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 //= link social_networking.js
 //= link social_networking.css
+//= link social_networking/_profile_icon_admin.png
```
Comment (eschlange) [inline spec/dummy/app/assets/config/manifest.js]: Were you able to test that this icon loaded locally in a host application? I was having trouble loading image resources through the engine manifests. I had to add the image paths to the host application manifest for several TFDEngine images.

Comment (michaelwehrley) [inline spec/dummy/app/assets/config/manifest.js]: good question!  On it!

Comment (michaelwehrley) [inline spec/dummy/app/assets/config/manifest.js]: @eschlange @ericcf I got it to work.  Thanks all!

**Comment by eschlange** [comment]
Nice additional tests.

**Pull request merged**